common/test.c

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright © 2017-2018 The Crust Firmware Authors.
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#include <debug.h>
+#include <test.h>
+
+int test_crust_boots(){
+
+	test("test: Boot: Pass");
```

Comment:
The `test:` here is redundant, because it will be added by `log()` in `debug.c`. Please follow the capitalization from the project documentation and reiterated in #18.

Reply:
done